app/controllers/videos_controller.rb

```diff
@@ -7,10 +7,30 @@ def index
 
   def show
     @video = Video.find(params[:id])
+    @review = Review.new
   end
 
   def search
     @videos = Video.search_by_title(params[:search])
   end
 
+  def review
```

Comment:
Yeah, I was thinking about that as I watched the solutions. I initially made the decision since only videos had reviews, and since reviews were nested as a member of videos. Is it generally frowned upon to deviate from CRUD?